---
layout:	page
subheadline:	Congratulations!
title: 	"Blair's spatial genome alignment (SGA) paper is published in Nature Biotechnology!"
teaser: "In spatial genomics, instead of ACGT genes are spatial positions (x, y, z). An ortholog of sequence aligners, SGA aligns these (x, y, z)'s to a reference polymer model of DNA, to see if they 'fit'."
breadcrumb: true	
tags:	
    - paper accepted	
categories:	
    - news	
image:	
    thumb: 	"2023-07-27-jia_spatial_genome_aligner_2023.png"
    title:	"2023-07-27-jia_spatial_genome_aligner_2023.png"
    caption_url: 	http://unsplash.com
---	
	
<b>Abstract</b>:
Multiplexed fluorescence in situ hybridization (FISH) is a widely used approach for analyzing three-dimensional genome organization, but it is challenging to derive chromosomal conformations from noisy fluorescence signals, and tracing chromatin is not straightforward. Here we report a spatial genome aligner that parses true chromatin signal from noise by aligning signals to a DNA polymer model. Using genomic distances separating imaged loci, our aligner estimates spatial distances expected to separate loci on a polymer in three-dimensional space. Our aligner then evaluates the physical probability observed signals belonging to these loci are connected, thereby tracing chromatin structures. We demonstrate that this spatial genome aligner can efficiently model chromosome architectures from DNA FISH data across multiple scales and be used to predict chromosome ploidies de novo in interphase cells. Reprocessing of previous whole-genome chromosome tracing data with this method indicates the spatial aggregation of sister chromatids in S/G2 phase cells in asynchronous mouse embryonic stem cells and provides evidence for extranumerary chromosomes that remain tightly paired in postmitotic neurons of the adult mouse cortex.
In spatial genomics, instead of ACGT genes are spatial positions (x, y, z). An ortholog of sequence aligners, SGA aligns these (x, y, z)'s to a reference polymer model of DNA, to see if they 'fit'.
> Full text can be accessed from the following [link](https://doi.org/10.1038/s41587-022-01568-9)
