---
layout: page
title: " MPE-seq a new method for the genome-wide analysis of chromatin structure."
breadcrumb: true
categories:
    - publication
## publication related information
pub:
    authors: " Haruhiko Ishii, James T. Kadonaga<sup>#</sup>,  Bing Ren<sup>#</sup>"
    journal: " Proceedings of the National Academy of Sciences of the United States of America"
    date: 2015-07-07
    doi:  10.1073/pnas.1424804112
    volume:  112
    pages:  E3457--3465
    number:  27
    abstract: " The analysis of chromatin structure is essential for the understanding of transcriptional regulation in eukaryotes. Here we describe methidiumpropyl-EDTA sequencing (MPE-seq), a method for the genome-wide characterization of chromatin  that involves the digestion of nuclei withMPE-Fe(II) followed by massively parallel sequencing. Like micrococcal nuclease (MNase), MPE-Fe(II) preferentially cleaves the linker DNA between nucleosomes. However, there are differences in the cleavage of nuclear chromatin by MPE-Fe(II) relative to MNase. Most notably, immediately upstream of the transcription start site of active promoters, we frequently observed nucleosome-sized (141-190 bp) and subnucleosome-sized (such as 101-140 bp) peaks of digested chromatin fragments with MPE-seq but not with MNase-seq. These peaks also correlate with the presence of core histones and could thus be due, at least in part, to noncanonical chromatin structures such as labile nucleosome-like particles that have been observed in other contexts. The subnucleosome-sized MPE-seq peaks exhibit a particularly distinct association with active promoters. In addition, unlike MNase, MPE-Fe(II) cleaves nuclear DNA  with little sequence bias. In this regard, we found that DNA sequences at RNA splice sites are hypersensitive to digestion by MNase but not by MPE-Fe(II). This phenomenon may have affected the analysis of nucleosome occupancy over exons. These findings collectively indicate that MPE-seq provides a unique and straightforward means for the genome-wide analysis of chromatin structure with minimal DNA sequence bias. In particular, the combined use of MPE-seq and MNase-seq enables the identification of noncanonical chromatin structures that are likely to be important for the regulation of gene expression.,"
---
