echo title2bib -f  A spatial genome aligner for resolving chromatin architectures from multiplexed DNA FISH
title2bib -f  A spatial genome aligner for resolving chromatin architectures from multiplexed DNA FISH
echo title2bib -f  Age-dependent Lamin changes induce cardiac dysfunction via dysregulation of cardiac transcriptional programs
title2bib -f  Age-dependent Lamin changes induce cardiac dysfunction via dysregulation of cardiac transcriptional programs
echo title2bib -f  KDM6A Loss Triggers an Epigenetic Switch That Disrupts Urothelial Differentiation and Drives Cell Proliferation in Bladder Cancer
title2bib -f  KDM6A Loss Triggers an Epigenetic Switch That Disrupts Urothelial Differentiation and Drives Cell Proliferation in Bladder Cancer
echo title2bib -f  The EN-TEx resource of multi-tissue personal epigenomes 
title2bib -f  The EN-TEx resource of multi-tissue personal epigenomes 
echo title2bib -f  Topologically associating domain boundaries are required for normal genome function
title2bib -f  Topologically associating domain boundaries are required for normal genome function
echo title2bib -f  Dynamic changes in P300 enhancers and enhancer-promoter contacts control mouse cardiomyocyte maturation
title2bib -f  Dynamic changes in P300 enhancers and enhancer-promoter contacts control mouse cardiomyocyte maturation
echo title2bib -f  Integrated analysis of single-cell chromatin state and transcriptome identified common vulnerability despite glioblastoma heterogeneity
title2bib -f  Integrated analysis of single-cell chromatin state and transcriptome identified common vulnerability despite glioblastoma heterogeneity
echo title2bib -f  SALL1 enforces microglia-specific DNA binding and function of SMADs to establish microglia identity
title2bib -f  SALL1 enforces microglia-specific DNA binding and function of SMADs to establish microglia identity
echo title2bib -f  Robust enhancer-gene regulation identified by single-cell transcriptomes and epigenomes
title2bib -f  Robust enhancer-gene regulation identified by single-cell transcriptomes and epigenomes
echo title2bib -f  Droplet-based single-cell joint profiling of histone modifications and transcriptomes
title2bib -f  Droplet-based single-cell joint profiling of histone modifications and transcriptomes
echo title2bib -f  SnapFISH: a computational pipeline to identify chromatin loops from multiplexed DNA FISH data
title2bib -f  SnapFISH: a computational pipeline to identify chromatin loops from multiplexed DNA FISH data
echo title2bib -f  SnapHiC-D: a computational pipeline to identify differential chromatin contacts from single-cell Hi-C data
title2bib -f  SnapHiC-D: a computational pipeline to identify differential chromatin contacts from single-cell Hi-C data
echo title2bib -f  Improved epicardial cardiac fibroblast generation from iPSCs
title2bib -f  Improved epicardial cardiac fibroblast generation from iPSCs
echo title2bib -f  Single nucleus multi-omics identifies human cortical cell regulatory genome diversity
title2bib -f  Single nucleus multi-omics identifies human cortical cell regulatory genome diversity
echo title2bib -f  The Hippo pathway mediates Semaphorin signaling
title2bib -f  The Hippo pathway mediates Semaphorin signaling
echo title2bib -f  Human-gained heart enhancers are associated with species-specific cardiac attributes
title2bib -f  Human-gained heart enhancers are associated with species-specific cardiac attributes
echo title2bib -f  Single-cell epigenome analysis reveals age-associated decay of heterochromatin domains in excitatory neurons in the mouse brain
title2bib -f  Single-cell epigenome analysis reveals age-associated decay of heterochromatin domains in excitatory neurons in the mouse brain
echo title2bib -f  Systematic discovery and functional dissection of enhancers needed for cancer cell fitness and proliferation
title2bib -f  Systematic discovery and functional dissection of enhancers needed for cancer cell fitness and proliferation
echo title2bib -f   Type 1 diabetes risk genes mediate pancreatic beta cell survival in response to proinflammatory cytokines
title2bib -f   Type 1 diabetes risk genes mediate pancreatic beta cell survival in response to proinflammatory cytokines
echo title2bib -f   Systematic Analysis of Transcription Factor Binding to Noncoding Variants in the Human Genome
title2bib -f   Systematic Analysis of Transcription Factor Binding to Noncoding Variants in the Human Genome
echo title2bib -f  Promoter-proximal CTCF-binding promotes Distal-enhancer dependent gene activation
title2bib -f  Promoter-proximal CTCF-binding promotes Distal-enhancer dependent gene activation
echo title2bib -f  Pancreatic progenitor epigenome maps prioritize type 2 diabetes risk genes with roles in development
title2bib -f  Pancreatic progenitor epigenome maps prioritize type 2 diabetes risk genes with roles in development
echo title2bib -f  KDM1A maintains genome-wide homeostasis of transcriptional enhancers
title2bib -f  KDM1A maintains genome-wide homeostasis of transcriptional enhancers
echo title2bib -f  ZNF143 mediates CTCF-bound promoter-enhancer loops required for murine hematopoietic stem and progenitor cell function
title2bib -f  ZNF143 mediates CTCF-bound promoter-enhancer loops required for murine hematopoietic stem and progenitor cell function
echo title2bib -f  Joint profiling of histone modifications and transcriptome in single cells from mouse cortex and hippocampus
title2bib -f  Joint profiling of histone modifications and transcriptome in single cells from mouse cortex and hippocampus
echo title2bib -f  SnapATAC: A Comprehensive Analysis Package for Single Cell ATAC-seq
title2bib -f  SnapATAC: A Comprehensive Analysis Package for Single Cell ATAC-seq
echo title2bib -f  A β-catenin-driven switch in TCF/LEF transcription factor binding to DNA target sites promotes commitment of mammalian nephron progenitor cells
title2bib -f  A β-catenin-driven switch in TCF/LEF transcription factor binding to DNA target sites promotes commitment of mammalian nephron progenitor cells
echo title2bib -f  Iterative single-cell multi-omic integration using online learning
title2bib -f  Iterative single-cell multi-omic integration using online learning
echo title2bib -f  Cardiac cell type-specific gene regulatory programs and disease risk association
title2bib -f  Cardiac cell type-specific gene regulatory programs and disease risk association
echo title2bib -f  CTCF mediates dosage- and sequence-context-dependent transcriptional insulation by forming local chromatin domains
title2bib -f  CTCF mediates dosage- and sequence-context-dependent transcriptional insulation by forming local chromatin domains
echo title2bib -f  SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data
title2bib -f  SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data
echo title2bib -f  Cell-autonomous inflammation of BRCA1-deficient ovarian cancers drives both tumor-intrinsic immunoreactivity and immune resistance via STING
title2bib -f  Cell-autonomous inflammation of BRCA1-deficient ovarian cancers drives both tumor-intrinsic immunoreactivity and immune resistance via STING
echo title2bib -f  Parallel characterization of cis-regulatory elements for multiple genes using CRISPRpath
title2bib -f  Parallel characterization of cis-regulatory elements for multiple genes using CRISPRpath
echo title2bib -f  A multimodal cell census and atlas of the mammalian primary motor cortex
title2bib -f  A multimodal cell census and atlas of the mammalian primary motor cortex
echo title2bib -f  An atlas of gene regulatory elements in adult mouse cerebrum
title2bib -f  An atlas of gene regulatory elements in adult mouse cerebrum
echo title2bib -f  A transcriptomic and epigenomic cell atlas of the mouse primary motor cortex
title2bib -f  A transcriptomic and epigenomic cell atlas of the mouse primary motor cortex
echo title2bib -f  Comparative cellular analysis of motor cortex in human
title2bib -f  Comparative cellular analysis of motor cortex in human
echo title2bib -f  DNA methylation atlas of the mouse brain at single-cell resolution
title2bib -f  DNA methylation atlas of the mouse brain at single-cell resolution
echo title2bib -f  A single-cell atlas of chromatin accessibility in the human genome
title2bib -f  A single-cell atlas of chromatin accessibility in the human genome
echo title2bib -f  Sequence logic at enhancers governs a dual mechanism of endodermal organ fate induction by FOXA pioneer factors
title2bib -f  Sequence logic at enhancers governs a dual mechanism of endodermal organ fate induction by FOXA pioneer factors
echo title2bib -f  CTCF mediates chromatin looping via N-terminal domain-dependent cohesin retention
title2bib -f  CTCF mediates chromatin looping via N-terminal domain-dependent cohesin retention
echo title2bib -f  Genome-wide association and multi-omic analyses reveal ACTN2 as a gene linked to heart failure
title2bib -f  Genome-wide association and multi-omic analyses reveal ACTN2 as a gene linked to heart failure
echo title2bib -f  An atlas of dynamic chromatin landscapes in mouse fetal development
title2bib -f  An atlas of dynamic chromatin landscapes in mouse fetal development
echo title2bib -f  Expanded Encyclopedias of DNA Elements in the Human and Mouse Genomes
title2bib -f  Expanded Encyclopedias of DNA Elements in the Human and Mouse Genomes
echo title2bib -f  Spatiotemporal DNA methylome dynamics of the developing mouse fetus
title2bib -f  Spatiotemporal DNA methylome dynamics of the developing mouse fetus
echo title2bib -f  The changing mouse embryo transcriptome at whole tissue and single-cell resolution
title2bib -f  The changing mouse embryo transcriptome at whole tissue and single-cell resolution
echo title2bib -f  Direct DNA crosslinking with CAP-C uncovers transcription-dependent chromatin organization at high resolution
title2bib -f  Direct DNA crosslinking with CAP-C uncovers transcription-dependent chromatin organization at high resolution
echo title2bib -f  resides within an enhancer subject to lineage- and disease-dependent epigenetic remodeling
title2bib -f  resides within an enhancer subject to lineage- and disease-dependent epigenetic remodeling
echo title2bib -f  Chromothripsis drives the evolution of gene amplification in cancer
title2bib -f  Chromothripsis drives the evolution of gene amplification in cancer
echo title2bib -f  FIREcaller: Detecting frequently interacting regions from Hi-C data
title2bib -f  FIREcaller: Detecting frequently interacting regions from Hi-C data
echo title2bib -f  Genomic encoding of transcriptional burst kinetics
title2bib -f  Genomic encoding of transcriptional burst kinetics
echo title2bib -f  Dynamic reorganization of the genome shapes the recombination landscape in meiotic prophase
title2bib -f  Dynamic reorganization of the genome shapes the recombination landscape in meiotic prophase
echo title2bib -f  MAPS: Model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments
title2bib -f  MAPS: Model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments
echo title2bib -f  Multi-platform discovery of haplotype-resolved structural variation in human genomes
title2bib -f  Multi-platform discovery of haplotype-resolved structural variation in human genomes
echo title2bib -f  NAD metabolic dependency in cancer is shaped by gene amplification and enhancer remodelling
title2bib -f  NAD metabolic dependency in cancer is shaped by gene amplification and enhancer remodelling
echo title2bib -f  Pancreatic islet chromatin accessibility and conformation reveals distal enhancer networks of type 2 diabetes risk
title2bib -f  Pancreatic islet chromatin accessibility and conformation reveals distal enhancer networks of type 2 diabetes risk
echo title2bib -f  DNA replication acts as an error correction mechanism to maintain centromere identity by restricting CENP-A to centromeres
title2bib -f  DNA replication acts as an error correction mechanism to maintain centromere identity by restricting CENP-A to centromeres
echo title2bib -f  Comparing 3D Genome Organization in Multiple Species Using Phylo-HMRF
title2bib -f  Comparing 3D Genome Organization in Multiple Species Using Phylo-HMRF
echo title2bib -f  Joint profiling of DNA methylation and chromatin architecture in single cells
title2bib -f  Joint profiling of DNA methylation and chromatin architecture in single cells
echo title2bib -f  Transcription Factor-Directed Re-wiring of Chromatin Architecture for Somatic Cell Nuclear Reprogramming toward trans-Differentiation
title2bib -f  Transcription Factor-Directed Re-wiring of Chromatin Architecture for Somatic Cell Nuclear Reprogramming toward trans-Differentiation
echo title2bib -f  Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells
title2bib -f  Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells
echo title2bib -f   A compendium of promoter-centered long-range chromatin interactions in the human genome
title2bib -f   A compendium of promoter-centered long-range chromatin interactions in the human genome
echo title2bib -f  Circular extrachromosomal DNA promotes oncogene expression and chromatin accessibility
title2bib -f  Circular extrachromosomal DNA promotes oncogene expression and chromatin accessibility
echo title2bib -f  An ultra high-throughput method for combined analysis of accessible chromatin and transcriptome in single cells
title2bib -f  An ultra high-throughput method for combined analysis of accessible chromatin and transcriptome in single cells
echo title2bib -f  Metabolic regulation of gene expression by histone lactylation
title2bib -f  Metabolic regulation of gene expression by histone lactylation
echo title2bib -f  Single-Cell Chromatin Analysis of Mammary Gland Development Reveals Cell-State Transcriptional Regulators and Lineage Relationships
title2bib -f  Single-Cell Chromatin Analysis of Mammary Gland Development Reveals Cell-State Transcriptional Regulators and Lineage Relationships
echo title2bib -f  Brain cell type-specific enhancer-promoter interactome maps and disease risk association
title2bib -f  Brain cell type-specific enhancer-promoter interactome maps and disease risk association
echo title2bib -f  Ranking of non-coding pathogenic variants and putative essential regions of the human genome
title2bib -f  Ranking of non-coding pathogenic variants and putative essential regions of the human genome
echo title2bib -f  Common DNA sequence variation influences 3-dimensional conformation of the human genome
title2bib -f  Common DNA sequence variation influences 3-dimensional conformation of the human genome
echo title2bib -f  Histone H3 lysine 4 monomethylation modulates long- range chromatin interactions at enhancers
title2bib -f  Histone H3 lysine 4 monomethylation modulates long- range chromatin interactions at enhancers
echo title2bib -f  Transcriptional regulatory control of mammalian nephron progenitors revealed by multi-factor cistromic analysis and genetic studies
title2bib -f  Transcriptional regulatory control of mammalian nephron progenitors revealed by multi-factor cistromic analysis and genetic studies
echo title2bib -f  Single-nucleus analysis of accessible chromatin in developing mouse forebrain reveals cell-type-specific transcriptional regulation
title2bib -f  Single-nucleus analysis of accessible chromatin in developing mouse forebrain reveals cell-type-specific transcriptional regulation
echo title2bib -f  The human noncoding genome defined by genetic diversity
title2bib -f  The human noncoding genome defined by genetic diversity
echo title2bib -f  Mapping of genomic EGFRvIII deletions in glioblastoma: insight into rearrangement mechanisms and biomarker development
title2bib -f  Mapping of genomic EGFRvIII deletions in glioblastoma: insight into rearrangement mechanisms and biomarker development
echo title2bib -f  a nucleosome destabilizing factor that facilitates transcription through 65 nucleosomes
title2bib -f  a nucleosome destabilizing factor that facilitates transcription through 65 nucleosomes
echo title2bib -f  Analysis of genetically diverse macrophages reveals local and domain-wide mechanisms that control transcription factor binding and function
title2bib -f  Analysis of genetically diverse macrophages reveals local and domain-wide mechanisms that control transcription factor binding and function
echo title2bib -f  The Hippo pathway effector proteins YAP and TAZ have both distinct and overlapping functions in the cell
title2bib -f  The Hippo pathway effector proteins YAP and TAZ have both distinct and overlapping functions in the cell
echo title2bib -f  RAP2 mediates mechanoresponses of the Hippo pathway
title2bib -f  RAP2 mediates mechanoresponses of the Hippo pathway
echo title2bib -f  Coordinated histone modifications and chromatin reorganization in a single cell revealed by FRET biosensors
title2bib -f  Coordinated histone modifications and chromatin reorganization in a single cell revealed by FRET biosensors
echo title2bib -f  Hyper-Editing of Cell-Cycle Regulatory and Tumor Suppressor RNA Promotes Malignant Progenitor Propagation
title2bib -f  Hyper-Editing of Cell-Cycle Regulatory and Tumor Suppressor RNA Promotes Malignant Progenitor Propagation
echo title2bib -f  Extrachromosomal oncogene amplification drives tumour evolution and genetic heterogeneity
title2bib -f  Extrachromosomal oncogene amplification drives tumour evolution and genetic heterogeneity
echo title2bib -f  Improved regulatory element prediction based on tissue-specific local epigenomic signatures
title2bib -f  Improved regulatory element prediction based on tissue-specific local epigenomic signatures
echo title2bib -f  A Scalable Epitope Tagging Approach for High Throughput ChIP-Seq Analysis
title2bib -f  A Scalable Epitope Tagging Approach for High Throughput ChIP-Seq Analysis
echo title2bib -f  Human centromeric CENP-A chromatin is a homotypic
title2bib -f  Human centromeric CENP-A chromatin is a homotypic
echo title2bib -f  Large-Scale Profiling Reveals the Influence of Genetic Variation on Gene Expression in Human Induced Pluripotent Stem Cells
title2bib -f  Large-Scale Profiling Reveals the Influence of Genetic Variation on Gene Expression in Human Induced Pluripotent Stem Cells
echo title2bib -f  A tiling-deletion-based genetic screen for cis-regulatory element identification in mammalian cells
title2bib -f  A tiling-deletion-based genetic screen for cis-regulatory element identification in mammalian cells
echo title2bib -f  HUGIn: Hi-C Unifying Genomic Interrogator
title2bib -f  HUGIn: Hi-C Unifying Genomic Interrogator
echo title2bib -f  High Resolution Mapping of Chromatin Conformation in Cardiac Myocytes Reveals Structural Remodeling of the Epigenome in Heart Failure
title2bib -f  High Resolution Mapping of Chromatin Conformation in Cardiac Myocytes Reveals Structural Remodeling of the Epigenome in Heart Failure
echo title2bib -f  Preventive Inhibition of Liver Tumorigenesis by Systemic Activation of Innate Immune Functions
title2bib -f  Preventive Inhibition of Liver Tumorigenesis by Systemic Activation of Innate Immune Functions
echo title2bib -f  Identification of H3K4me1-associated proteins at mammalian enhancers
title2bib -f  Identification of H3K4me1-associated proteins at mammalian enhancers
echo title2bib -f  A new class of temporarily phenotypic enhancers identified by CRISPR/Cas9 mediated genetic screening
title2bib -f  A new class of temporarily phenotypic enhancers identified by CRISPR/Cas9 mediated genetic screening
echo title2bib -f  Broad histone H3K4me3 domains in mouse oocytes modulate maternal to zygotic transition
title2bib -f  Broad histone H3K4me3 domains in mouse oocytes modulate maternal to zygotic transition
echo title2bib -f  Genome-wide compendium and functional assessment of in vivo heart enhancers
title2bib -f  Genome-wide compendium and functional assessment of in vivo heart enhancers
echo title2bib -f  A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome
title2bib -f  A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome
echo title2bib -f  Mapping of long-range chromatin interactions by proximity ligation-assisted ChIP-seq
title2bib -f  Mapping of long-range chromatin interactions by proximity ligation-assisted ChIP-seq
echo title2bib -f  DNA binding by FOXP3 domain-swapped dimer suggests mechanisms of long-range chromosomal interactions
title2bib -f  DNA binding by FOXP3 domain-swapped dimer suggests mechanisms of long-range chromosomal interactions
echo title2bib -f  Chromatin Architecture Reorganization during Stem Cell Differentiation
title2bib -f  Chromatin Architecture Reorganization during Stem Cell Differentiation
echo title2bib -f  Integrative analysis of haplotype-resolved epigenomes across human tissues
title2bib -f  Integrative analysis of haplotype-resolved epigenomes across human tissues
echo title2bib -f  Integrative analysis of 111 reference human epigenomes
title2bib -f  Integrative analysis of 111 reference human epigenomes
echo title2bib -f  Developmental Competence Encoded in Enhancer Chromatin
title2bib -f  Developmental Competence Encoded in Enhancer Chromatin
echo title2bib -f  An alternative pluripotent state confers interspecies chimaeric competency
title2bib -f  An alternative pluripotent state confers interspecies chimaeric competency
echo title2bib -f  a new method for the genome-wide analysis of chromatin structure
title2bib -f  a new method for the genome-wide analysis of chromatin structure
echo title2bib -f  Human body epigenome maps reveal noncanonical DNA methylation variation
title2bib -f  Human body epigenome maps reveal noncanonical DNA methylation variation
echo title2bib -f  CRISPR Inversion of CTCF Sites Alters Genome Topology and Enhancer/Promoter Function
title2bib -f  CRISPR Inversion of CTCF Sites Alters Genome Topology and Enhancer/Promoter Function
echo title2bib -f  Comparative analyses of CTCF and BORIS occupancies uncover two distinct classes of CTCF binding genomic regions
title2bib -f  Comparative analyses of CTCF and BORIS occupancies uncover two distinct classes of CTCF binding genomic regions
echo title2bib -f   Complete haplotype phasing of the MHC and KIR loci with targeted HaploSeq
title2bib -f   Complete haplotype phasing of the MHC and KIR loci with targeted HaploSeq
echo title2bib -f  EGFR Mutation Promotes Glioblastoma through Epigenome and Transcription Factor Network Remodeling
title2bib -f  EGFR Mutation Promotes Glioblastoma through Epigenome and Transcription Factor Network Remodeling
echo title2bib -f  Identification of global regulators of T-helper cell lineage specification
title2bib -f  Identification of global regulators of T-helper cell lineage specification
echo title2bib -f  Lysine 2-hydroxyisobutyrylation is a widely distributed active histone mark
title2bib -f  Lysine 2-hydroxyisobutyrylation is a widely distributed active histone mark
echo title2bib -f  Regulation of DNA methylation turnover at LTR retrotransposons and imprinted loci by the histone methyltransferase Setdb1
title2bib -f  Regulation of DNA methylation turnover at LTR retrotransposons and imprinted loci by the histone methyltransferase Setdb1
echo title2bib -f  Tissue-specific SMARCA4 binding at active and repressed regulatory elements during embryogenesis
title2bib -f  Tissue-specific SMARCA4 binding at active and repressed regulatory elements during embryogenesis
echo title2bib -f  Distinct and predictive histone lysine acetylation patterns at promoters
title2bib -f  Distinct and predictive histone lysine acetylation patterns at promoters
echo title2bib -f  5mC oxidation by Tet2 modulates enhancer activity and timing of transcriptome reprogramming during differentiation
title2bib -f  5mC oxidation by Tet2 modulates enhancer activity and timing of transcriptome reprogramming during differentiation
echo title2bib -f  Comparative encyclopedia of DNA elements in the mouse genome
title2bib -f  Comparative encyclopedia of DNA elements in the mouse genome
echo title2bib -f  Topologically-associating domains are stable units of replication-timing regulation
title2bib -f  Topologically-associating domains are stable units of replication-timing regulation
echo title2bib -f  CRISPR Reveals a Distal Super-Enhancer Required for Sox2 Expression in Mouse Embryonic Stem Cells
title2bib -f  CRISPR Reveals a Distal Super-Enhancer Required for Sox2 Expression in Mouse Embryonic Stem Cells
echo title2bib -f  HAT cofactor Trrap Maintains Self-renewal and Restricts Differentiation of Embryonic Stem Cells
title2bib -f  HAT cofactor Trrap Maintains Self-renewal and Restricts Differentiation of Embryonic Stem Cells
echo title2bib -f  Bayesian Inference of Spatial Organizations of Chromosomes
title2bib -f  Bayesian Inference of Spatial Organizations of Chromosomes
echo title2bib -f  Tet-mediated covalent labelling of 5-methylcytosine for its genome-wide detection and sequencing
title2bib -f  Tet-mediated covalent labelling of 5-methylcytosine for its genome-wide detection and sequencing
echo title2bib -f  RFECS: A Random-Forest Based Algorithm for Enhancer Identification From Chromatin State
title2bib -f  RFECS: A Random-Forest Based Algorithm for Enhancer Identification From Chromatin State
echo title2bib -f  Epigenomic Analysis of Multi-lineage Differentiation of Human Embryonic Stem Cells
title2bib -f  Epigenomic Analysis of Multi-lineage Differentiation of Human Embryonic Stem Cells
echo title2bib -f  Comparative annotation of functional regions in the human genome using epigenomic data
title2bib -f  Comparative annotation of functional regions in the human genome using epigenomic data
echo title2bib -f  Temporal transcriptional response to ethylene gas drives growth hormone cross-regulation in Arabidopsis
title2bib -f  Temporal transcriptional response to ethylene gas drives growth hormone cross-regulation in Arabidopsis
echo title2bib -f  Global Chromatin State Analysis Reveals Lineage-Specific Enhancers during the Initiation of Human T helper 1 and T helper 2 Cell Polarization
title2bib -f  Global Chromatin State Analysis Reveals Lineage-Specific Enhancers during the Initiation of Human T helper 1 and T helper 2 Cell Polarization
echo title2bib -f  Epigenetic memory at embryonic enhancers identified from adult tissue methylomes
title2bib -f  Epigenetic memory at embryonic enhancers identified from adult tissue methylomes
echo title2bib -f  A High-resolution Map of Signaling-dependent
title2bib -f  A High-resolution Map of Signaling-dependent
echo title2bib -f  STAR: an integrated solution to management and visualization of sequencing data
title2bib -f  STAR: an integrated solution to management and visualization of sequencing data
echo title2bib -f  Fine tuning of craniofacial morphology by distant-acting enhancers
title2bib -f  Fine tuning of craniofacial morphology by distant-acting enhancers
echo title2bib -f  Whole-genome haplotype reconstruction using proximity-ligation and shotgun sequencing
title2bib -f  Whole-genome haplotype reconstruction using proximity-ligation and shotgun sequencing
echo title2bib -f  N6-methyladenosine-dependent regulation of messenger RNA stability
title2bib -f  N6-methyladenosine-dependent regulation of messenger RNA stability
echo title2bib -f  Systematic mapping of occluded genes by cell fusion reveals prevalence and stability of cis- mediated silencing in somatic cells
title2bib -f  Systematic mapping of occluded genes by cell fusion reveals prevalence and stability of cis- mediated silencing in somatic cells
echo title2bib -f  Cohesin and CTCF differentially affect chromatin architecture and gene expression in human cells
title2bib -f  Cohesin and CTCF differentially affect chromatin architecture and gene expression in human cells
echo title2bib -f  involving mediator and competitive interactions of Gdown1 and TFIIF with Pol II
title2bib -f  involving mediator and competitive interactions of Gdown1 and TFIIF with Pol II
echo title2bib -f  Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome
title2bib -f  Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome
echo title2bib -f  Polycomb-like 3 promotes polycomb repressive complex 2 binding to CpG islands and embryonic stem cell self-renewal
title2bib -f  Polycomb-like 3 promotes polycomb repressive complex 2 binding to CpG islands and embryonic stem cell self-renewal
echo title2bib -f  Topological domains in mammalian genomes identified by analysis of chromatin interactions
title2bib -f  Topological domains in mammalian genomes identified by analysis of chromatin interactions
echo title2bib -f  Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome
title2bib -f  Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome
echo title2bib -f  Global identification of transcriptional regulators of pluripotency and differentiation in embryonic stem cells
title2bib -f  Global identification of transcriptional regulators of pluripotency and differentiation in embryonic stem cells
echo title2bib -f  Combined gene expression and DNA occupancy profiling identifies potential therapeutic targets of t
title2bib -f  Combined gene expression and DNA occupancy profiling identifies potential therapeutic targets of t
echo title2bib -f  A map of the cis-regulatory sequences in the mouse genome
title2bib -f  A map of the cis-regulatory sequences in the mouse genome
echo title2bib -f  CBX3 regulates efficient RNA processing genome-wide
title2bib -f  CBX3 regulates efficient RNA processing genome-wide
echo title2bib -f  HiCNorm: removing biases in Hi-C data via Poisson regression
title2bib -f  HiCNorm: removing biases in Hi-C data via Poisson regression
echo title2bib -f  Progressive degeneration of human neural stem cells caused by pathogenic LRRK2
title2bib -f  Progressive degeneration of human neural stem cells caused by pathogenic LRRK2
echo title2bib -f  Tet-assisted bisulfite sequencing of 5-hydroxymethylcytosine
title2bib -f  Tet-assisted bisulfite sequencing of 5-hydroxymethylcytosine
echo title2bib -f  Hotspots of aberrant epigenomic reprogramming in human induced pluripotent stem cells
title2bib -f  Hotspots of aberrant epigenomic reprogramming in human induced pluripotent stem cells
echo title2bib -f  Multiple enhancers in the 9p21 gene desert contain regulatory variants associated with Coronary Artery Disease and Type 2 Diabetes risk
title2bib -f  Multiple enhancers in the 9p21 gene desert contain regulatory variants associated with Coronary Artery Disease and Type 2 Diabetes risk
echo title2bib -f 1 and C/EBPα synergistically program distinct response to NF-κB activation through establishing monocyte specific enhancers
title2bib -f 1 and C/EBPα synergistically program distinct response to NF-κB activation through establishing monocyte specific enhancers
echo title2bib -f  A cis-regulatory map of the Drosophila genome
title2bib -f  A cis-regulatory map of the Drosophila genome
echo title2bib -f  Endogenous retroviruses and neighboring genes are coordinately repressed by LSD1/KDM1A
title2bib -f  Endogenous retroviruses and neighboring genes are coordinately repressed by LSD1/KDM1A
echo title2bib -f  Distinct patterns of somatic alterations in a lymphoblastoid and a tumor genome derived from the same individual
title2bib -f  Distinct patterns of somatic alterations in a lymphoblastoid and a tumor genome derived from the same individual
echo title2bib -f  SERPINB13 is a novel RUNX1 target gene
title2bib -f  SERPINB13 is a novel RUNX1 target gene
echo title2bib -f  Integrating 5-hydroxymethylcytosine into the epigenomic landscape of human embryonic stem cells
title2bib -f  Integrating 5-hydroxymethylcytosine into the epigenomic landscape of human embryonic stem cells
echo title2bib -f  Identification of 67 histone marks and histone lysine crotonylation as a new type of histone modification
title2bib -f  Identification of 67 histone marks and histone lysine crotonylation as a new type of histone modification
echo title2bib -f  Dynamic Chromatin States in Human ES Cells Reveal Potential Regulatory Sequences and Genes Involved in Pluripotency
title2bib -f  Dynamic Chromatin States in Human ES Cells Reveal Potential Regulatory Sequences and Genes Involved in Pluripotency
echo title2bib -f  Global DNA Hypomethylation Coupled to Repressive Chromatin Domain Formation and Gene Silencing in Breast Cancer
title2bib -f  Global DNA Hypomethylation Coupled to Repressive Chromatin Domain Formation and Gene Silencing in Breast Cancer
echo title2bib -f  Genome-wide prediction of transcription factor binding sites using an integrated model
title2bib -f  Genome-wide prediction of transcription factor binding sites using an integrated model
echo title2bib -f  Distinct epigenomic landscapes of pluripotent and lineage-committed human cells
title2bib -f  Distinct epigenomic landscapes of pluripotent and lineage-committed human cells
echo title2bib -f Comparison of sequencing-based methods to profile DNA methylation and identification of monoallelic epigenetic modifications
title2bib -f Comparison of sequencing-based methods to profile DNA methylation and identification of monoallelic epigenetic modifications
echo title2bib -f  p16INK4a-mediated suppression of telomerase in normal and malignant human breast cells
title2bib -f  p16INK4a-mediated suppression of telomerase in normal and malignant human breast cells
echo title2bib -f  ChIP-Seq identification of weakly conserved heart enhancers
title2bib -f  ChIP-Seq identification of weakly conserved heart enhancers
echo title2bib -f   An assessment of histone-modification antibody quality
title2bib -f   An assessment of histone-modification antibody quality
echo title2bib -f  Systematic detection of putative tumor suppressor genes through the combined use of exome and transcriptome sequencing
title2bib -f  Systematic detection of putative tumor suppressor genes through the combined use of exome and transcriptome sequencing
echo title2bib -f  Identification of Functional Elements and Regulatory Circuits by Drosophila modENCODE
title2bib -f  Identification of Functional Elements and Regulatory Circuits by Drosophila modENCODE
echo title2bib -f  Histone Modifications at Enhancers Reflect Global Cell Type-Specific Gene Expression
title2bib -f  Histone Modifications at Enhancers Reflect Global Cell Type-Specific Gene Expression
echo title2bib -f  ChIP-seq Accurately Predicts Tissue-Specific Activity of Enhancers
title2bib -f  ChIP-seq Accurately Predicts Tissue-Specific Activity of Enhancers
echo title2bib -f  An integrated approach to identifying cis-regulatory modules in the human genome
title2bib -f  An integrated approach to identifying cis-regulatory modules in the human genome
echo title2bib -f  Discovery and Annotation of Functional Chromatin Signatures in the Human Genome
title2bib -f  Discovery and Annotation of Functional Chromatin Signatures in the Human Genome
echo title2bib -f  Human DNA methylomes at base resolution show widespread epigenomic differences
title2bib -f  Human DNA methylomes at base resolution show widespread epigenomic differences
echo title2bib -f  Genome-wide mapping of active promoters in mouse embryonic stem cells and adult organs
title2bib -f  Genome-wide mapping of active promoters in mouse embryonic stem cells and adult organs
echo title2bib -f  Genome-wide Mapping of Allele-specific Protein-DNA Interactions in Human Cells
title2bib -f  Genome-wide Mapping of Allele-specific Protein-DNA Interactions in Human Cells
echo title2bib -f  Systematic evaluation of variability in ChIP-chip experiments using predefined DNA targets
title2bib -f  Systematic evaluation of variability in ChIP-chip experiments using predefined DNA targets
echo title2bib -f  in the regulation of NF-kappa B dependent inflammatory genes: Relevance to diabetes and inflammation
title2bib -f  in the regulation of NF-kappa B dependent inflammatory genes: Relevance to diabetes and inflammation
echo title2bib -f  ChromaSig: A Probabilistic Approach to Finding Common Chromatin Signatures in the Human Genome
title2bib -f  ChromaSig: A Probabilistic Approach to Finding Common Chromatin Signatures in the Human Genome
echo title2bib -f  Prediction of Regulatory Elements in Mammalian Genomes Using Chromatin Signatures
title2bib -f  Prediction of Regulatory Elements in Mammalian Genomes Using Chromatin Signatures
echo title2bib -f  ChIP-chip data model and analysis
title2bib -f  ChIP-chip data model and analysis
echo title2bib -f   Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome
title2bib -f   Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome
echo title2bib -f  Analysis of the vertebrate insulator protein CTCF binding in the human genome
title2bib -f  Analysis of the vertebrate insulator protein CTCF binding in the human genome
echo title2bib -f  The ENCODE pilot project: Identification and analysis of functional elements in 1% of the human genome
title2bib -f  The ENCODE pilot project: Identification and analysis of functional elements in 1% of the human genome
echo title2bib -f  a gene linked to speech and language in developing human brain
title2bib -f  a gene linked to speech and language in developing human brain
echo title2bib -f  Identification and Characterization of Cell Type-Specific and Ubiquitous Chromatin Regulatory Structures in the Human Genome
title2bib -f  Identification and Characterization of Cell Type-Specific and Ubiquitous Chromatin Regulatory Structures in the Human Genome
echo title2bib -f  Positive inter-regulation between beta-catenin/T cell factor-4 signaling and endothelin-1 signaling potentiates proliferation and survival of prostate cancer cells
title2bib -f  Positive inter-regulation between beta-catenin/T cell factor-4 signaling and endothelin-1 signaling potentiates proliferation and survival of prostate cancer cells
echo title2bib -f  Mnt-deficient mammary glands exhibit impaired involution and tumors with characteristics of Myc overexpression
title2bib -f  Mnt-deficient mammary glands exhibit impaired involution and tumors with characteristics of Myc overexpression
echo title2bib -f   β-catenin activates endothelin-1 in colon cancer cells
title2bib -f   β-catenin activates endothelin-1 in colon cancer cells
echo title2bib -f  Direct isolation and identification of promoters in the human genome
title2bib -f  Direct isolation and identification of promoters in the human genome
echo title2bib -f  A high-resolution map of active promoters in the human genome
title2bib -f  A high-resolution map of active promoters in the human genome
echo title2bib -f  A Global Regulatory Role for c-myc in Burkitt
title2bib -f  A Global Regulatory Role for c-myc in Burkitt
